spec/fixture/project1.json

```diff
@@ -0,0 +1,64 @@
+{
+  "vm": {
+    "hostname": "acme.dev",
+    "synced_folders":[{
+      "_id": "100",
+      "type": "rsync",
+      "host": "./workspace/angular-hello-world",
+      "guest": "/home/vagrant/workspace/angular-hello-world",
+      "rsync__exclude": [".git", ".idea/", "node_modules/", "bower_components/", ".npm/"]
+    }, {
+      "_id": "100",
+      "type": "virtual_box",
+      "host": "workspace/angular-hello-world/node_modules",
+      "guest": "/home/vagrant/workspace/angular-hello-world/node_modules",
+      "mount_options": [
+        "dmode=775",
+        "fmode=755"
+      ]
+    }, {
+      // enable this to sync /dist back and forth to the host automatically
+      "_id": "101",
+      "type": "virtual_box",
+      "host": "workspace/angular-hello-world/dist",
+      "guest": "/home/vagrant/workspace/angular-hello-world/dist",
+      "mount_options": [
+        "dmode=775",
+        "fmode=755"
+      ]
+    }]
+  },
+
+  "provisioners": [{
+    "_id": "0",
+    "_ua_cookbooks_path": [
+      "workspace/angular-hello-world/chef/main-cookbooks"
```

Comment:
it's best practices to have "dev-setup" on the project =>  "workspace/angular-hello-world/dev-setup/chef/main-cookbooks"